qt/Session.cc

```diff
@@ -589,6 +591,7 @@ std::vector<std::string_view> const& Session::getKeyNames(TorrentProperties prop
             TR_KEY_fileStats, //
             TR_KEY_haveUnchecked, //
             TR_KEY_honorsSessionLimits, //
+            TR_KEY_labels, //
```

Comment:
Same question here, plus the field is already in `DetailInfoKeys`  so I'm a llittle confused

Reply:
moved to `DetailInfoKeys`. Now calling `session_.refreshDetailInfo()` after labels edit